aio_pika/pool.py

```diff
@@ -30,7 +31,7 @@ class PoolInvalidStateError(RuntimeError):
     pass
 
 
-class Pool:
+class Pool(Generic[T]):
```

Comment:
Is it necessary here? I don't see any usage of `Pool[T]`.

Reply:
It's used by the user of the aio-pika library.

```
class Messenger:
    connection_pool: Pool[aio_pika.Connection]
    channel_pool: Pool[aio_pika.Channel]
```